source/reference/menus.rst

```diff
@@ -0,0 +1,105 @@
+===================
+ Menus - Reference
+===================
+
+.. seealso::
+
+   :doc:`Documentation on menus </customization/menus>`
+      Explains how menus work and what you can do
+
+
+File Format
+===========
+
+=============  ===========================================
+**Format**     JSON (with comments)
+
+**Extension**  ``.sublime-menu``
+
+**Name**       One out of the list of available menus.
+               See :ref:`menu-types` for the complete list
```

Comment:
explicit ==> complete list of names

I think there's another case above I didn't point out.